pkg/chartutil/create.go

```diff
@@ -151,7 +151,7 @@ const defaultIgnore = `# Patterns to ignore when building packages.
 const defaultIngress = `{{- if .Values.ingress.enabled -}}
 {{- $fullName := include "<CHARTNAME>.fullname" . -}}
 {{- $svcPort := .Values.service.port -}}
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Could we have some logic to handle both the old and new `apiVersion`? This way create will work across k8s versions. #6448 added it to dev-v3 but I don't see it on v2 or even in v3 anymore.

Reply:
I would have been of the same thinking but there was a good [response](https://github.com/helm/helm/pull/6490#discussion_r331752270) from @unguiculus on this in #6490. I am easy either way but why are we putting this in place when we support k8s n-1 anyways?

Reply:
The charts repository has had an n-1 support philosophy. In practice it hasn't really followed that lately. Does helm itself have something? I ask because many of the public clouds have Kubernetes n-2 as their latest stable. That includes EKS, Google Cloud, and Azure. The Google Cloud current default is 1.13. `helm create` not being able to create charts that run in the latest stable releases from the largest public clouds seems like a problem. Is there a good case for n-1 support?

Reply:
Ok, I put back the check in place for k8s < 1.14.